client/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## 1.X.X (yyyy-mm-dd)
+
+- Basic Yseop Engine model based type completion
```

Comment:
Use an hyphen (“Yseop Engine model-based type completion”) or, since this isn't quite sufficient to make it pleasing to read, write something like “Basic type names completion, based on Yseop Engine's model”. If you agree.

Reply:
Ok, I'll do the changes.